### PR TITLE
Upgrade Ethereum Client for Median

### PR DESCRIPTION
**Pull request by teghnet**

### CODE REVIEW

File: config.hcl [hunk 1]
```diff
@@ -56,28 +68,32 @@ ethereum {
     }
   }
 
-  client "ethereum" {
-    rpc_urls = try(env.CFG_ETH_RPC_URLS == "" ? [] : split(",", env.CFG_ETH_RPC_URLS), [
-      "https://eth.public-rpc.com"
-    ])
-    chain_id     = tonumber(try(env.CFG_ETH_CHAIN_ID, "1"))
-    ethereum_key = "default"
+  dynamic "client" {
```
Comment (deep-quality-dev) [inline config.hcl:71]: Needs to update `cmd/**/README.md`

Comment (teghnet) [reply config.hcl]: Not really, this "dynamic" block is a feature of HCL.
It is still the same client.

File: pkg/rpcsplitter/server.go [hunk 2]
```diff
@@ -547,7 +547,7 @@ func (s *server) call(
 					s.log.
 						WithField("name", n).
 						WithField("method", method).
-						WithField("args", args).
+						// WithField("args", args).
```
Comment (MDobak) [inline pkg/rpcsplitter/server.go:550]: Why did you decide to comment this out?

**Review by MDobak — APPROVED** [review]
It looks good to me. I really like the way you refactored the config files.

File: pkg/rpcsplitter/server.go (hunk 2 above)
Comment (teghnet) [reply pkg/rpcsplitter/server.go:550]: It was too verbose.
We need to go over logging in general to see where we can make improvements.

**Pull request merged**